src/soso/strategies/spase.py

```diff
@@ -41,22 +41,42 @@ def __init__(self, file: str, **kwargs: dict):
         super().__init__(metadata=etree.parse(file))
         self.file = file
         self.schema_version = get_schema_version(self.metadata)
+        self.namespaces = {"spase": "http://www.spase-group.org/data/schema"}
         self.kwargs = kwargs
 
     def get_id(self) -> None:
-        dataset_id = None
+        """schema:identifier: spase:ResourceID)"""
+        dataset_id = self.metadata.findtext(
+            ".//spase:NumericalData/spase:ResourceID", namespaces=self.namespaces
+        )
         return delete_null_values(dataset_id)
 
     def get_name(self) -> None:
-        name = None
+        """schema:description: spase:ResourceHeader/ResourceName"""
+        name = self.metadata.findtext(
+            ".//spase:NumericalData/spase:ResourceHeader/spase:ResourceName",
+            namespaces=self.namespaces,
+        )
         return delete_null_values(name)
 
     def get_description(self) -> None:
-        description = None
+        """schema:description: spase:ResourceHeader/Description"""
+        description = self.metadata.findtext(
+            ".//spase:NumericalData/spase:ResourceHeader/spase:Description",
+            namespaces=self.namespaces,
+        )
         return delete_null_values(description)
 
     def get_url(self) -> None:
```

Comment:
Update the return type from `None` to `str` to match the result. If the return value can be `None`, because the SPASE element is not required, then define the return type as `Union[str, None]`.